src/pages/SatelliteManifestPage/SatelliteManifestPage.tsx

```diff
@@ -1,28 +1,42 @@
-import React from 'react';
+import React, { FC, useContext } from 'react';
 import { withRouter } from 'react-router-dom';
 import Main from '@redhat-cloud-services/frontend-components/Main';
 import PageHeader from '@redhat-cloud-services/frontend-components/PageHeader';
 import { PageHeaderTitle } from '@redhat-cloud-services/frontend-components/PageHeader';
 import SatelliteManifestPanel from '../../components/SatelliteManifestPanel';
 import useSatelliteManifests from '../../hooks/useSatelliteManifests';
 import Unavailable from '@redhat-cloud-services/frontend-components/Unavailable';
+import { NoSatelliteManifests } from '../../components/emptyState';
+import NotAuthorized from '@redhat-cloud-services/frontend-components/NotAuthorized';
+import { Processing } from '../../components/emptyState';
+import UserContext from '../../components/Authentication/UserContext';
 
-const SatelliteManifestPage = () => {
+const SatelliteManifestPage: FC = () => {
   const { isLoading, error, data } = useSatelliteManifests();
+  const { user } = useContext(UserContext);
 
   return (
-    <React.Fragment>
+    <>
       <PageHeader>
         <PageHeaderTitle title="Satellite Manifests" />
         <p>Export subscriptions to your on-premise subscription management application</p>
       </PageHeader>
       <Main>
         <>
-          {!error && <SatelliteManifestPanel isLoading={isLoading} data={data} />}
+          {isLoading && <Processing />}
+
+          {!isLoading && data?.length > 0 && <SatelliteManifestPanel data={data} />}
+
+          {!isLoading && data?.length === 0 && user.isOrgAdmin === true && <NoSatelliteManifests />}
+
+          {!isLoading && data?.length === 0 && user.isOrgAdmin === false && (
```

Comment:
Aha good call; I'd had to add that to the NoSatelliteManifests component check due to a bug where it shows when it's not supposed to, but it appears unnecessary for the NotAuthorized component

Reply:
Won't it render both the table and the not authorized component now if there are results and the person is non-admin?

My confusion wasn't really with the technical aspects of the code, but with why we would rather non-admin users see "not authorized" than just seeing a table with no results.  It seems misleading since non-admins do seem to be authorized to access the table.

Reply:
Aha ok yes I had thought there was that scenario.  

In the ticket, https://issues.redhat.com/browse/TEAMNADO-1991, it says, per Tramaine's note, that a no access message should be presented.  I also agree that it should show a no permissions issue - if they need to use this but don't have permissions, it is probably better that they realize it's an issue with permissions, rather than an issue with no manifests existing in their account, no?  LMK if there's something I'm missing, or if we should check with Tramaine or anyone else.

Reply:
OK so I made an update to show the empty table instead of NoPermissions when user is not org_admin and no manifests are found.

I noticed that the verbiage of the NoSearchResults component ("no results match the filter criteria...") didn't make sense for when a user has no manifests and hasn't been filtering.  So I created an EmptyTable component where title, body, etc. can be passed into it, and changed the language for this scenario to reflect that no manifests were found under the account.

Abstracting this out may be a bit overkill and we could just have a <NoSearchResults /> and a <NoManifestsFound /> component, if we don't anticipate reusing this a lot in the future.

LMK thoughts.

Reply:
I was thinking about that same issue of the no search results text not making much sense, but decided not to worry about it now because this story seems to be expanding a bit and this is something we would normally be provided with content guidance on.

The EmptyTable component may be a bit much.  Even with just these two states, they don't have the same needs.  I don't think the state representing there being no manifests at all necessarily even needs a button, and another state may have multiple buttons or an external link, etc.

Reply:
OK sounds good; I just went ahead and went back to having two separate components, NoSearchResults and NoManifestsFound.  I removed the button from NoManifestsFound as well.

Reply:
Okay.  Seems good for now.